data-fusion/s1_flooding_visualisation/mosaic.js

```diff
@@ -0,0 +1,34 @@
+//VERSION=3
+function setup() {
+    return {
+      input: [{
+          datasource: "CUSTOM",
```

Comment:
The script is named sentinel-2 mosaic. I think it would be better to name the data source here

Reply:
CUSTOM is the default label of the data source in the Browser when setting up a data fusion script. I prefer to avoid renaming from the default because it adds complexit.